app/src/main/java/com/sunasterisk/englishflashcard/ui/splashscreen/SplashScreen.kt

```diff
@@ -0,0 +1,33 @@
+package com.sunasterisk.englishflashcard.ui.splashscreen
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.sunasterisk.englishflashcard.R
+import com.sunasterisk.englishflashcard.ui.main.MainActivity
+
+
+class SplashScreen : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+        val time: Thread = object : Thread() {
+            override fun run() {
+                try {
+                    sleep(3000)
+                } catch (e: Exception) {
+                } finally {
+                    val intent = Intent(applicationContext,MainActivity::class.java)
+                    startActivity(intent)
```

Comment:
chưa đúng convention rồi em
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#211-argument-in-activity-and-fragment